File: dotnet/Multiplayer/Rooms/Actions/ActionCompleteResponse.cs
using Michitai.Multiplayer.Errors;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Michitai.Multiplayer.Rooms.Actions
{
    /// <summary>
    /// Response returned when an action is successfully marked as complete.
    /// Only the host can complete actions.
    /// </summary>
    public class ActionCompleteResponse : ApiResponse<ERoomActionsCompleteError>
    {
        /// <summary>
        /// Confirmation message for the action completion.
        /// </summary>
        public string Message { get; set; } = string.Empty;
    }
}

File: java/src/main/java/com/michitai/multiplayer/rooms/actions/ActionCompleteResponse.java
package com.michitai.multiplayer.rooms.actions;

import com.fasterxml.jackson.annotation.JsonProperty;
import com.michitai.multiplayer.ApiResponse;

/**
 * Response returned when an action is successfully completed.
 */
public class ActionCompleteResponse extends ApiResponse {
    @JsonProperty("message")
    private String message;

    @JsonProperty("action_id")
    private String actionId;

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getActionId() {
        return actionId;
    }

    public void setActionId(String actionId) {
        this.actionId = actionId;
    }
}
